homeassistant/components/lutron/binary_sensor.py

```diff
@@ -0,0 +1,60 @@
+"""Support for Lutron Powr Savr motion sensors."""
+import logging
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+
+from . import LUTRON_CONTROLLER, LUTRON_DEVICES, LutronDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the Lutron motion sensors."""
+    devs = []
+    for (area_name, device) in hass.data[LUTRON_DEVICES]["binary_sensor"]:
+        dev = LutronMotionSensor(area_name, device, hass.data[LUTRON_CONTROLLER])
+        devs.append(dev)
+
+    add_entities(devs, True)
+
+
+class LutronMotionSensor(LutronDevice, BinarySensorDevice):
+    """Representation of a Lutron Occupancy Group.
+
+    The Lutron integration API reports "occupancy groups" rather than
+    motion sensors. If two motion sensors are in the same room, they're
+    reported as a single occupancy group.
+    """
+
+    @property
+    def is_on(self):
+        """Return true if the binary sensor is on."""
+        from pylutron import OccupancyGroup
+
+        # Error cases will end up treated as unoccupied.
+        _LOGGER.debug(
```

Comment:
We don't want side effects in entity state properties. Please remove the logging.